### PR TITLE
feat(dev-tools-rquest-resize): dev tools details panel can be resized horizontally via a drag handle

### PR DESCRIPTION
**Pull request by sachin-thakur-bruno**
### Description

This PR targets the **Details panel can be resized horizontally via a drag handle**

The details panel that opens when clicking a network request had a fixed width with no way to resize it.

### Changes

**`useResizablePanel` hook** (`hooks/useResizablePanel/index.js`)
- New reusable hook encapsulating drag-to-resize logic
- Supports `direction: 'left' | 'right'` — controls which drag direction expands the panel
- Clamps width between `minWidth` and `maxWidth`
- Calls `onResizeEnd` with the final width on mouseup
- 12 unit tests covering both directions, clamping, and edge cases

**`Console/index.js`**
- Replaced inline drag state/refs/handlers with `useResizablePanel({ direction: 'right' })`
- Resized width persists within the session via Redux (`logs` slice)

**`Console/StyledWrapper.js`**
- Added `.details-panel-wrapper` and `.details-drag-handle` styles
- Drag handle matches the sidebar's resize UX (`sidebar.dragbar.border` / `sidebar.dragbar.activeBorder` theme tokens)

**`RequestDetailsPanel/StyledWrapper.js`**
- Removed hardcoded `min-width: 400px`, `max-width: 600px`, `width: 40%`
- Replaced with `width: 100%` — parent wrapper now controls width

**`providers/ReduxStore/slices/logs.js`**
- Added `requestDetailsPanelWidth: 400` to `initialState`
- Added `updateRequestDetailsPanelWidth` reducer

https://github.com/user-attachments/assets/8233d372-e91c-4b2a-85bb-100cae09e912

#### Contribution Checklist:

- [ ] **I've used AI significantly to create this pull request**
- [ ] **The pull request only addresses one issue or adds one feature.**
- [ ] **The pull request does not introduce any breaking changes**
- [ ] **I have added screenshots or gifs to help explain the change if applicable.**
- [ ] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [ ] **Create an issue and link to the pull request.**

Note: Keeping the PR small and focused helps make it easier to review and merge. If you have multiple changes you want to make, please consider submitting them as separate pull requests.

#### Publishing to New Package Managers

Please see [here](../publishing.md) for more information.

## Summary by CodeRabbit

* **New Features**
  * Request Details panel in the Network console is now resizable with a draggable divider; width is persisted between sessions.
  * Visual improvements to the details panel layout and drag handle for clearer affordance.

* **Tests**
  * Added comprehensive tests covering drag-to-resize behavior, width clamping, and resize end handling.

### CODE REVIEW

File: packages/bruno-app/src/hooks/useResizablePanel/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  const handleMouseUp = () => {
+    if (isDragging.current) {
+      if (onResizeEnd) onResizeEnd(currentWidth.current);
+    }
+    isDragging.current = false;
+  };
+
+  useEffect(() => {
+    document.addEventListener('mousemove', handleMouseMove);
```
Comment (sid-bruno) [inline packages/bruno-app/src/hooks/useResizablePanel/index.js:60]:
```suggestion
    // Note: tying the events to the document instead of the parent to avoid fast movement 
    // from breaking the flow state of dragging 
    document.addEventListener('mousemove', handleMouseMove);
```

**Review by sid-bruno — CHANGES_REQUESTED** [review]

**Review by sid-bruno — APPROVED** [review]
